CommandLine/libEGM/egm-events.cpp

```diff
@@ -42,7 +42,12 @@ void LoadObjectEvents(const fs::path& fPath, google::protobuf::Message *m, const
 
 void WriteObjectEvents(const fs::path& fPath, const google::protobuf::RepeatedPtrField<buffers::resources::Object_EgmEvent>& events, const EventData* event_data) {
   for (auto &e : events) {
-    auto event = event_data->get_event(e.id(), {e.arguments().begin(), e.arguments().end()});
+    std::vector<std::string> args;
+    args.reserve(e.arguments_size());
+    for (const auto& arg : e.arguments()) {
+      args.push_back(std::string(arg));
+    }
+    auto event = event_data->get_event(std::string(e.id()), args);
```

Comment:
I don't think this change is necessary. It's possible the vector argument needs to be constructed explicitly now, though it is not marked explicit by the standard afaik.

I guess `e.id()` *does* need to be wrapped in `string()`, but I think it's better (as in most cases in this PR) to just modify `get_event()` to accept a `string_view`.